### DISCUSSION TITLE
Fuel station: which building type

### DISCUSSION THREAD

**Discussion by araccaine** [streetcomplete/StreetComplete#2933] [Q&A] [answered]
I was asked to specify the building type for a fuel station:
![ScreenshotSC.png](https://user-images.githubusercontent.com/75312500/120117495-c71c4400-c18d-11eb-9600-69a2fff39c3c.png)

But which type should I choose? Industrial building?
![ScreenshotSC2.png](https://user-images.githubusercontent.com/75312500/120117543-fc289680-c18d-11eb-8f77-1beb315f9453.png)

It is just a regular fuel station you find everywhere.

Thank you for your help :)

**Comment by HolgerJeromin** [comment, marked as answer]
I would tag it as retail

**Reply by araccaine** [reply]
Just as a clarification: the fuel station consists of two parts like in this example: ![Fuel station example](http://smartenergytec.com/wp-content/uploads/2015/05/project-gas-station.jpg).  

So one would tag the retail part on the left as *retail*, and the "roof-part" where the fuel pumps are also as retail?
thanks in advance :)

**Reply by matkoniecz** [reply]
I would tag part with shop as shop/retail, and roof part as a roof.
